tools/build_tar/buildtar.go

```diff
@@ -330,7 +328,107 @@ func (f *tarFile) addTar(toAdd string) error {
 }
 
 func (f *tarFile) addDeb(toAdd string) error {
-	return fmt.Errorf("addDeb unimplemented")
+	pkgDataFound, pkgMetadataFound := false, false
+	file, err := os.Open(toAdd)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+	tr := ar.NewReader(bufio.NewReader(file))
```

Comment:
nit: I think `tr` was short for `tar reader` in `addTar`, so somethink like `reader` is probably a better name here.

Reply:
done